apps/jenkins/jenkins/sbox-intsvc/jenkins-azure-vm-agent.yaml

```diff
@@ -30,73 +99,13 @@ spec:
                     maxVirtualMachinesLimit: 10
```

Comment:
this needs moving to the VM templates instead, probably 10 for production and 3 daily builds or something for sandbox

Reply:
moved to VM template defs